lld/wasm/LTO.cpp

```diff
@@ -59,6 +59,7 @@ static lto::Config createConfig() {
   c.DisableVerify = ctx.arg.disableVerify;
   c.DiagHandler = diagnosticHandler;
   c.OptLevel = ctx.arg.ltoo;
+  c.CPU = getCPUStr();
```

Comment:
Yup this looks like this is already present in the ELF version of `createConfig` .. I wonder why we didn't have it here before.

Can we write a test for this?

Reply:
i noticed this while building an application. i don't have smaller test cases right now.